Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -567,7 +567,7 @@ withStmt :
 
 // Special forms with special syntax, only available in VBA reports or VB6 forms and pictureboxes.
 lineSpecialForm : expression whiteSpace ((STEP whiteSpace?)? tuple)? MINUS (STEP whiteSpace?)? tuple whiteSpace? (COMMA whiteSpace? expression)? whiteSpace? (COMMA whiteSpace? lineSpecialFormOption)?;
-circleSpecialForm : (expression whiteSpace? DOT whiteSpace?)? CIRCLE whiteSpace (STEP whiteSpace?)? tuple (whiteSpace? COMMA whiteSpace? expression)+;
+circleSpecialForm : (expression whiteSpace? DOT whiteSpace?)? CIRCLE (whiteSpace STEP)? whiteSpace? tuple (whiteSpace? COMMA whiteSpace? expression)+;
```

Comment:
As it stands, (and IIUC) the new rule allows for `Circle(1,3) 2`, note the missing space before the opening parens of the tuple. Is that going to be alright?